app/services/login/user_verifier.rb

```diff
@@ -101,7 +101,8 @@ def create_user_verification
       UserVerification.create!(type => identifier,
                                user_account: existing_user_account || UserAccount.new(icn:),
                                backing_idme_uuid:,
-                               verified_at:)
+                               verified_at:,
+                               locked: existing_user_account ? validate_linked_user_verifications : false)
```

Comment:
can you move this conditional into the function itself (like an initial line, `return unless existing_user_account`)

also maybe update the name to mention that it's looking for locked accounts, and that it returns a boolean, something like `linked_user_verification_is_locked?` or something like that

Reply:
Could you add a log too? Like: 'locking due to existing locked user verification' or something like that (should include the user_verification identifier and type as well in the log)? Would be good to track how often this happens